packages/react-composites/src/composites/CallWithChatComposite/CallWithChatPane.tsx

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { IStackStyles, Stack } from '@fluentui/react';
+import { ParticipantMenuItemsCallback, useTheme, _DrawerMenu, _DrawerMenuItemProps } from '@internal/react-components';
+import React, { useMemo, useState } from 'react';
+import { CallAdapter } from '../CallComposite';
+import { CallAdapterProvider } from '../CallComposite/adapter/CallAdapterProvider';
+import { ChatAdapter, ChatComposite, ChatCompositeProps } from '../ChatComposite';
+import { AvatarPersonaDataCallback } from '../common/AvatarPersona';
+import {
+  paneBodyContainer,
+  scrollableContainer,
+  scrollableContainerContents
+} from '../common/styles/ParticipantContainer.styles';
+import { BasicHeader } from './BasicHeader';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import { LocalAndRemotePIPInModal, LocalAndRemotePIPInModalStyles } from './LocalAndRemotePIPInModal';
+import { PeoplePaneContent } from './PeoplePaneContent';
+import { drawerContainerStyles } from './styles/CallWithChatCompositeStyles';
+import { TabHeader } from './TabHeader';
+/* @conditional-compile-remove(file-sharing) */
+import { FileSharingOptions } from '../ChatComposite';
+
+/**
+ * Pane that is used to store chat and people for CallWithChat composite
+ * @private
+ */
+export const CallWithChatPane = (props: {
+  chatCompositeProps: Partial<ChatCompositeProps>;
+  callAdapter: CallAdapter;
+  chatAdapter: ChatAdapter;
+  onClose: () => void;
+  onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  onChatButtonClicked: () => void;
+  onPeopleButtonClicked: () => void;
+  modalLayerHostId: string;
+  activePane: 'chat' | 'people' | 'none';
```

Comment:
nit `'chat' | 'people' | 'none'` is listed in a couple places, consider making a type 
`type CallWithChatPaneOptions = 'none' | 'chat' | 'people'`